components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/JsGraphBuilder.java

```diff
@@ -152,6 +152,7 @@ public JsGraphBuilder createWith(String script) {
                     (LoadExecutor) this::loadLocalLibrary);
             engineBindings.put("exit", (RestrictedFunction) this::exitFunction);
             engineBindings.put("quit", (RestrictedFunction) this::quitFunction);
+            restrictDefaultFunctions(engine);
```

Comment:
shall we have exit and quit functions also under this?

Reply:
done